app/brave_generated_resources.grd

```diff
@@ -162,15 +162,28 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_PROFILES_OPEN_TOR_PROFILE_BUTTON" desc="Button in the avatar menu bubble view to open a Tor window.">
         Open Tor Window
       </message>
-      <message name="IDS_NEW_OFFTHERECORD_WINDOW_TOR" desc="The text label of a menu item to open a new off-the-record window with Tor.">
-        New private window with Tor
-      </message>
-      <message name="IDS_CONTENT_CONTEXT_OPENLINKTOR" desc="The name of the open a link in private window with Tor command">
-        Open link in private window with Tor
-      </message>
-      <message name="IDS_CONTENT_CONTEXT_OPENLINKTOR_INAPP" desc="The name of the open a link in private window with Tor command">
-        Open link in Brave private window with Tor
-      </message>
+      <if expr="use_titlecase">
```

Comment:
Where is this set?

Reply:
ah - found it: `src/tools/grit/grit_rule.gni`
```
# Mac and iOS want Title Case strings.
use_titlecase_in_grd_files = is_mac || is_ios
if (use_titlecase_in_grd_files) {
  grit_defines += [
    "-D",
    "use_titlecase",
  ]
}
```

Reply:
use_titlecase is set on macOS and iOS as they are the platforms which have this norm for menus